macOS/DuckDuckGo/Suggestions/ViewModel/SuggestionViewModel.swift

```diff
@@ -142,30 +142,28 @@ struct SuggestionViewModel: Equatable {
         }
     }
 
-    var suffix: String {
+    var suffix: String? {
         switch suggestion {
         // for punycoded urls display real url as a suffix
         case .website(url: let url) where url.toString(forUserInput: userStringValue, decodePunycode: false) != self.string:
-            return " – " + url.toString(decodePunycode: false, dropScheme: true, dropTrailingSlash: true)
+            return url.toString(decodePunycode: false, dropScheme: true, dropTrailingSlash: true)
 
         case .phrase, .unknown, .website:
-            return ""
-        case .openTab(title: _, url: let url) where url.isDuckURLScheme:
-            return " – " + UserText.duckDuckGo
-        case .openTab(title: _, url: let url) where url.isDuckDuckGoSearch:
-            return " – " + UserText.duckDuckGoSearchSuffix
+            return nil
+        case .openTab(title: _, url: let url, _) where url.isDuckURLScheme:
+            return UserText.duckDuckGo
+        case .openTab(title: _, url: let url, _) where url.isDuckDuckGoSearch:
+            return UserText.duckDuckGoSearchSuffix
         case .historyEntry(title: _, url: let url, allowedInTopHits: _),
              .bookmark(title: _, url: let url, isFavorite: _, allowedInTopHits: _),
-             .openTab(title: _, url: let url):
+             .openTab(title: _, url: let url, _):
             if url.isDuckDuckGoSearch {
-                return " – \(UserText.searchDuckDuckGoSuffix)"
+                return UserText.searchDuckDuckGoSuffix
             } else {
-                return " – " + url.toString(decodePunycode: true,
-                                              dropScheme: true,
-                                              dropTrailingSlash: true)
+                return url.toString(decodePunycode: true, dropScheme: true, dropTrailingSlash: true)
             }
         case .internalPage:
-            return " – " + UserText.duckDuckGo
+            return UserText.duckDuckGo
```

Comment:
" – " adding moved to `SuggestionTableCellView.swift` to match the suffix(or subtitle) to Windows implementation